apollo-gradle-plugin/src/test/testProject/android/buildScriptFixtures/flavored.gradle

```diff
@@ -0,0 +1,60 @@
+buildscript {
+  repositories {
+    jcenter()
+  }
+
+  dependencies {
+    classpath 'com.android.tools.build:gradle:2.3.0-beta3'
+  }
+}
+
+plugins {
+  id 'com.android.application'
+  id 'com.apollographql.android'
+}
+
+android {
+  compileSdkVersion 25
+  buildToolsVersion '25.0.2'
```

Comment:
We can write those through a groovy util method. I suppose that would make updating those values less painful.